examples/taproot-psbt.rs

```diff
@@ -0,0 +1,556 @@
+
+    let tx_hex_string = encode::serialize_hex(&generate_bip86_key_spend_tx(
+      &secp,
+      // The master extended private key from the descriptor in step 4
+      ExtendedPrivKey::from_str(BENEFACTOR_XPRIV_STR)?,
+      // Set these fields with valid data for the UTXO from step 5 above
+      UTXO_1,
+      vec![
+            TxOut {
+                value: amount_to_send_in_sats,
+                script_pubkey: to_address.script_pubkey(),
+            },
+            TxOut {
+                value: change_amount,
+                script_pubkey: change_address.script_pubkey(),
+            },
+        ],
+    )?);
+    println!("\nYou should now be able to broadcast the following transaction: \n\n{tx_hex_string}",);
+
+    println!("\nEND EXAMPLE 1\n");
+    println!("----------------\n");
+
+    println!("START EXAMPLE 2 - Script path spending of inheritance UTXO\n");
+    
+    let beneficiary = BeneficiaryWallet::new(ExtendedPrivKey::from_str(BENEFICIARY_XPRIV_STR)?)?;
+
+    let mut benefactor = BenefactorWallet::new(
+        ExtendedPrivKey::from_str(BENEFACTOR_XPRIV_STR)?,
+        beneficiary.master_xpub()
+    )?;
+    let (tx, psbt) = benefactor.create_inheritance_funding_tx(1000, UTXO_2)?;
+    let tx_hex = encode::serialize_hex(&tx);
+
+    println!("Inheritance funding tx hex:\n\n{}", tx_hex);
+    // You can now broadcast the transaction hex:
+    // bt sendrawtransaction ...
+    //
+    // And mine a block to confirm the transaction:
+    // bt generatetoaddress 1 $(bt-benefactor getnewaddress '' 'bech32m')
+
+    let spending_tx = beneficiary.spend_inheritance(psbt, 1000, to_address, 1000)?;
+    let spending_tx_hex = encode::serialize_hex(&spending_tx);
+    println!("\nInheritance spending tx hex:\n\n{}", spending_tx_hex);
+    // If you try to broadcast now, the transaction will be rejected as it is timelocked.
+    // First mine 900 blocks so we're sure we are over the 1000 block locktime:
+    // bt generatetoaddress 900 $(bt-benefactor getnewaddress '' 'bech32m')
+    // Then broadcast the transaction with `bt sendrawtransaction ...`
+
+    println!("\nEND EXAMPLE 2\n");
+    println!("----------------\n");
+
+    println!("START EXAMPLE 3 - Key path spending of inheritance UTXO\n");
+
+    let beneficiary = BeneficiaryWallet::new(ExtendedPrivKey::from_str(BENEFICIARY_XPRIV_STR)?)?;
+
+    let mut benefactor = BenefactorWallet::new(
+        ExtendedPrivKey::from_str(BENEFACTOR_XPRIV_STR)?,
+        beneficiary.master_xpub()
+    )?;
+    let (tx, _) = benefactor.create_inheritance_funding_tx(1000, UTXO_2)?;
+    let tx_hex = encode::serialize_hex(&tx);
+
+    println!("Inheritance funding tx hex:\n\n{}", tx_hex);
+    // You can now broadcast the transaction hex:
+    // bt sendrawtransaction ...
+    //
+    // And mine a block to confirm the transaction:
+    // bt generatetoaddress 1 $(bt-benefactor getnewaddress '' 'bech32m')
+
+    let spending_tx_hex = encode::serialize_hex(&spending_tx);
```

Comment:
```suggestion
    let spending_tx = beneficiary.spend_inheritance(psbt, 1200, &to_address, 1000)?;
    let spending_tx_hex = encode::serialize_hex(&spending_tx);
```

Note I used 1200 instead of 1000 otherwise one cannot verify the locktime if example two has already been done.

Reply:
We could use a const for the fee as well so as not to confuse it with locktime?